src/gl/implementation/gl-objects-buffers.adb

```diff
@@ -29,6 +29,15 @@ package body GL.Objects.Buffers is
 
    Current_Buffers : Buffer_Maps.Map;
 
+   package Transform_Buffer_Maps is new Ada.Containers.Indefinite_Hashed_Maps
+      (Key_Type     => Low_Level.Enums.Buffer_Kind,
+       Element_Type => Transform_Buffer'Class,
+       Hash         => Hash,
+       Equivalent_Keys => Low_Level.Enums."=");
+   use type Transform_Buffer_Maps.Cursor;
+
+   Current_Transform_Buffers : Transform_Buffer_Maps.Map;
```

Comment:
I don't see why this would be needed.

Reply:
`Indefinite_Hashed_Maps` changed to `Indefinite_Ordered_Maps`.
`Indefinite`  seems to be needed to allow `Element_Type => Transform_Buffer'Class`